starter-code/lib/viking.js

```diff
@@ -19,13 +19,77 @@
 //   FootViking: FootViking
 // };
 
-function Soldier () {}
+function Soldier () {
+  this.health = 300;
+  this.strength = 150;
+  this.attack = function attack () {};
+  this.receiveDamage = function damage () {
+    var damage = 50;
+    this.health = this.health - damage;
+  };
+}
 
-function Viking () {}
+Viking.prototype = Object.create(Soldier.prototype);
 
-function Saxon () {}
+Viking.prototype.constructor = Viking;
 
-function War () {}
+function Viking (name, health, strength) {
+  this.name = name;
+  Soldier.call(this, health, strength);
+  this.receiveDamage = function (health) {
+    if (health >= this.health) {
+      return (name + ' has died in act of combat');
+    } else {
+    this.health = this.health - health;
+    return this.name + ' has received ' + health + ' points of damage';
+    }
+  };
+  this.battleCry = function battleCry () {
+    return 'Odin Owns You All!';
+  };
+}
+
+Saxon.prototype = Object.create(Soldier.prototype);
+
+Saxon.prototype.constructor = Saxon;
+
+function Saxon (name, health, strength) {
+  Soldier.call(this, health, strength);
+  this.health = 60;
+  this.strength = 25;
+  this.receiveDamage = function (damage) {
+    if (health >= this.health) {
```

Comment:
`if (this.health <= 0`
makes more sense b/c this will check if the guy is alive or dead